serf/snapshot_test.go

```diff
@@ -391,6 +391,7 @@ func TestSnapshotter_leave_rejoin(t *testing.T) {
 }
 
 func TestSnapshotter_slowDiskNotBlockingEventCh(t *testing.T) {
+	t.Skip("Flaky test")
```

Comment:
@banks This is a test you wrote a while back. It's constantly flaking and I'm wondering if it's a perf regression or an issue with the test environment (it flake in circle as much as GHA). 

In any case, I think this is more a benchmark test and we will probably need to write it using a different approach. 
What do you think? is it safe to disable it for the time being (or even remove it) or this is a regression that we need to spend more time investigating?

Reply:
👍 

```
        // I typically see this complete in around
	// 115ms on my machine so this should give plenty of headroom for slower CI
	// environments while still being low enough that actual disk IO would
	// reliably blow it.
```

:D I guess I was wrong then! I think it was a more than just a benchmark in the sense that I wanted to verify that our implementation actually didn't allow disk IO to block gossip and effectively break the cluster but yeah see the commentary in https://github.com/hashicorp/serf/pull/526 - we assumed this was lenient enough for CI but if it's not then disabling seems OK?